InriaMuseum/InriaMuseum.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Inria Museum</title>
+    <link rel="stylesheet" type="text/css" href="InriaMuseum.css"> 
+    <script>
+
+
+        var current_page; //global variable
+        var previous_page;
+        function handleButtonClick(event) { //event listener for buttons
+            console.log(`ct handleButtonClick ${event}`);
+            console.log(`event.target.id ${event.target.id}`);
+
+            if(event.target.id == ""){
+                const parentElementId = event.target.parentElement ?
+                event.target.parentElement.id : 'No parent element';
+                console.log(`event.target.id ${event.target.parentElement.id}`)
+                changePage(parentElementId);
+            }else{
+                changePage(event.target.id);
+            }
+        }
+        
+        function changePage(ElementId) {
+            console.log(`fct changePage ${ElementId}`);
+            switch (ElementId) {
+                case "button_start_fr":
+                    current_page='launching.svg';
+                    break;
+                case "button_active": 
+                    current_page='map.svg';
+                    break;
+                case "button_suite":
+                    current_page='select_music.svg'
+                    break;
+                case "button_music1" :
+                    music_1();
+                    break;
+
+                case "button_music2" :
+                    music_2();
+                    break;
+                case "button1":
+                    current_page='map.svg';
+                    break;
+                case "button2":
+                    current_page='map.svg';
+                    break;
+                case "button3":
+                    current_page='map.svg';
+                    break;
+                case "button4":
+                    current_page='map.svg';
+                    break;
+                    
+                case "button_copyright":
+                    previous_page = current_page;
+                    current_page='credits.svg';
+                    break;
+                case "button_leave":
+                    current_page = previous_page;
+                    break;
+                default :
+                    console.log(`default ${ElementId} `); 
+                    break;
+            }
+            loadSVG(current_page);
+            checkPage();
+        }
+
+        function music_1(){
```

Comment:
why having specific functions for `music_1` and `music_2`? Please put the handling diercting in the switch/case statemen above.